miner/proposal_builder_test.go

```diff
@@ -145,9 +147,10 @@ func expectMeshHash(hash types.Hash32) expectOpt {
 	}
 }
 
+const epochExpectCounter = types.EpochID(3)
```

Comment:
This should probably always be `epoch+1` and `epoch` should again vary between tests (or `// nolint`)